applications/StructuralMechanicsApplication/python_scripts/adaptative_remeshing_structural_mechanics_implicit_dynamic_solver.py

```diff
@@ -45,8 +35,7 @@ def __init__(self, model, custom_settings):
 
     def AddVariables(self):
         super(AdaptativeRemeshingImplicitMechanicalSolver, self).AddVariables()
-        if not missing_meshing_dependencies:
-            self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.NODAL_H)
```

Comment:
was this approved by the StructuralMechanics team? @philbucher ?

Reply:
It is an independent solver

Reply:
to me ok, this is in the remeshing. If it needed there then ok